src/ig/IGExporter.ts

```diff
@@ -405,6 +407,77 @@ export class IGExporter {
     });
   }
 
+  /**
+   * Adds any user provided resource files
+   * This includes definitions in:
+   * capabilities, extensions, models, operations, profiles, resources, vocabulary, examples
+   *
+   * @param {string} igPath - the path where the IG is exported to
+   */
+  private addPredefinedResources(igPath: string): void {
+    const pathEnds = [
+      'capabilities',
+      'extensions',
+      'models',
+      'operations',
+      'profiles',
+      'resources',
+      'vocabulary',
+      'examples' // Must come last in case examples are of other resources
+    ];
+    for (const pathEnd of pathEnds) {
+      const dirPath = path.join(this.igDataPath, 'input', pathEnd);
+      if (fs.existsSync(dirPath)) {
+        const files = fs.readdirSync(dirPath);
+        for (const file of files) {
+          let resourceJSON: InstanceDefinition;
+          try {
+            resourceJSON = JSON.parse(fs.readFileSync(path.join(dirPath, file), 'utf-8').trim());
+          } catch (e) {
+            logger.error(`Invalid input file: ${path.join(dirPath, file)}`);
+            continue;
+          }
+          if (resourceJSON.resourceType == null || resourceJSON.id == null) {
+            logger.error(`Resource at ${path.join(dirPath, file)} must define resourceType and id`);
+            continue;
+          }
+          const resource: ImplementationGuideDefinitionResource = {
+            reference: {
+              reference: `${resourceJSON.resourceType}/${resourceJSON.id}`
+            },
+            name: resourceJSON.title ?? resourceJSON.id,
+            description: resourceJSON.description
+          };
+          if (pathEnd === 'examples') {
+            const exampleUrl = resourceJSON.meta?.profile?.find(
+              url =>
+                this.fhirDefs.fishForFHIR(url, Type.Profile) ?? this.pkg.fish(url, Type.Profile)
+            );
+            if (exampleUrl) {
+              resource.exampleCanonical = exampleUrl;
+            } else {
+              resource.exampleBoolean = true;
+            }
+          } else {
+            resource.exampleBoolean = false;
+          }
+          this.ig.definition.resource.push(resource);
```

Comment:
IIRC, we agreed with Keith that any resources explicitly put in the `ig-data` should _overwrite_ FSH-generated resources (if it's, in fact, the same resource).  So `push`ing on the `resource` array will just create duplicate resource entries instead of overwriting the old one.  I think we're going to need to search through the `resource` array to see if the resource already exists and then act accordingly.